doc/mp-init.md

```diff
@@ -163,14 +153,36 @@ mark. If your editor automatically includes the byte-order mark when saving to U
 | license_type              | Management Packs made using the VMware Aria Operations Integration SDK do not support licensing.                                                                                                                                                                                                                                   |
 
 <sup>1</sup> This property is required.
 
 <sup>2</sup> Localized fields are specified in the property files located in the `resources/` directory. See [Adding Localization](adding_content.md#adding-localization).
 
+### venv (directory)
+
+A Python virtual environment for the project. The directory name is prefixed with __venv__ followed by the display name of the management pack. This virtual environment is created to allow users to use different versions of the SDK for every project.
+
+**NOTE:** This directory is excluded from version control. Users cloning the project will have to create their own venv directory([ venv documentaion](https://docs.python.org/3/library/venv.html)).
+
+### app (directory)
+By default, this directory contains a template adapter. The template adapter collects several objects and metrics from the container that the adapter
+is running in, and can be used as a starting point for creating a new adapter. The template adapter uses [vmware-aria-operations-integration-sdk-lib](https://pypi.org/project/vmware-aria-operations-integration-sdk-lib/) to streamline the process of building adapter.
+The template adapter has comments throughout the code to help new users understand the process of creating their own adapter using the existing code. For additional guidance creating adapters see our [Walkthroughs](../README.md#walkthroughs) section.
+
+### conf (directory)
+The conf directory contains configuration files used by VMware Aria Operations to create a management pack instance.
+
+- describe.xml: An XML configuration file that defines the object model for an adapter, along with semantic definitions for use in data analysis and management.
+- describeSchema.xsd: A file that defines what elements and attributes may appear in the `describe.xml` document. This file does not need to be included in a Management Pack but is useful for validating that the `describe.xml` file is error-free.
+
+### content (directory)
+Contains all the content included in a management pack such as: alerts, dashboards, reports, recommendations, symptoms, and traversal specs. To learn more about adding content, consult the [Adding Content](adding_content.md) guidej
+
+### resources (directory)
+The resources directory contains localization file(s), by default `resources.properties`. This file specifies the mapping of `nameKey` attribute of elements inside the `conf/describe.xml`. For more information see [Adding Localization](adding_content.md#adding-localization).
```

Comment:
The top-level 'resources' directory has the localization file(s) for the manifest.txt file, not describe. The describe's resources folder is in the 'conf' directory next to the describe.xml file itself.